src/persist-client/src/operators/shard_source.rs

```diff
@@ -702,19 +643,30 @@ mod tests {
             let until = Antichain::new();
 
             let (probe, _token) = worker.dataflow::<u64, _, _>(|scope| {
-                let (stream, token) = shard_source::<String, String, u64, _, _>(
-                    scope,
-                    "test_source",
-                    persist_clients,
-                    location,
-                    shard_id,
-                    None, // No explicit as_of!
-                    until,
-                    None,
-                    Arc::new(<std::string::String as mz_persist_types::Codec>::Schema::default()),
-                    Arc::new(<std::string::String as mz_persist_types::Codec>::Schema::default()),
-                    |_fetch| true,
-                );
+                let (stream, token) = scope.scoped("hybrid", |scope| {
+                    let transformer = move |_, descs: &Stream<_, _>, _| {
```

Comment:
This is the default transformer so you can replace it with `None` below

Reply:
this is the only way i found that could fill in the `DT` type parameter :(

Reply:
Does that mean that the `DT` parameter should not be an `Option`?